File: C++/13-调整数组顺序使奇数位于偶数前面.cpp
class Solution {
public:
    void reOrderArray(vector<int> &array) {
        int len = array.size();
        int i, j;
        for(i = 0; i < len; i++){
            for(j = len - 1; j > i; j--){
                if((array[j] % 2 == 1) && (array[j - 1] % 2 == 0)){
                    int t = array[j];
                    array[j] = array[j - 1];
                    array[j - 1] = t;
                }
            }
        }
    }
};

File: Python/13-调整数组顺序使奇数位于偶数前面.py
class Solution:
    def reOrderArray(self, array):
        # write code here
        n = len(array)
        for i in range(n):
            for j in range(n - 1, i, -1):
                if array[j] % 2 == 1 and array[j - 1] % 2 == 0:
                    array[j], array[j - 1] = array[j - 1], array[j]
        return array
